CHANGELOG.md

```diff
@@ -22,6 +22,13 @@ Canonical reference for changes, improvements, and bugfixes for Boundary.
   catalog. ([PR](https://github.com/hashicorp/boundary/pull/5137) and
   [PR](https://github.com/hashicorp/boundary-plugin-aws/pull/49))
 
+* Improved MinIO storage plugin compatibility with other services by dropping
+  the checksum headers in `PutObject`.
+  ([PR](https://github.com/hashicorp/boundary-plugin-minio/pull/23))
+
+* Improved MinIO storage plugin error messages.
```

Comment:
I think we can drop this entry and just keep the first one.

Reply:
Yep - Updated https://github.com/hashicorp/boundary/compare/f72f98fcd870145e632626f3d6a31025d6c77959..15c7990501663a101673542c0bafd593c3fb4fc2 👍